src/styles/blocks/stars__star.css

```diff
@@ -0,0 +1,16 @@
+.stars__star {
+  margin: 0 2px;
```

Comment:
it will be better to add only margin left or right and then reset it to 0 on first-child or last-child :)